content/en/blog/2023/kubecon-eu.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Join us for OpenTelemetry Talks and Activities at Kubecon EU 2023
+linkTitle: Kubecon EU '23
+date: 2023-03-30
+spelling:
+  cSpell:ignore Severin Neumann Pavol Loffay Kowall Aiven Vider Xiaochun Alolita
+  Sharma Dyla Benedikt Bongartz Oliveira Pathak
+author: '[Severin Neumann](https://github.com/svrnm)'
+---
+
+The OpenTelemetry project maintainers, and members of the governance committee
+and technical committee are excited to be at [Kubecon EU][] in Amsterdam from
+April 18 - 21, 2023!
+
+You can meet us at the [project booth][], listen to our talks or learn with us
+during workshops. We are looking forward to hear from you, how you are using
+OpenTelemetry or contributing to the project.
+
+Below you will find a list of talks and other activities related to
+OpenTelemetry.
+
+## Kubecon Talks and Maintainer Sessions
+
+- **[Jaeger: The Future with OpenTelemetry and Metrics](https://sched.co/1HySf)**<br>
+  by Pavol Loffay, Red Hat & Jonah Kowall, Aiven<br> Wednesday, April 19 •
+  11:55 - 12:30
+- **[Using OpenTelemetry for Application Security, with a Real Life Example](https://sched.co/1HyYT)**<br>
+  by Ron Vider, Oxeye<br> Wednesday, April 19 • 11:55 - 12:30
+- **[Understand Systems with OpenTelemetry: A Hybrid Telemetry Data Backend](https://sched.co/1Hyb2)**<br>
+  by Ran Xu, Huawei & Xiaochun Yang, Northeastern University<br> Wednesday,
+  April 19 • 14:30 - 15:05
+- **[OTel Me About Metrics: A Metrics 101 Crash Course](https://sched.co/1Hya1)**<br>
+  by Reese Lee, New Relic<br> Wednesday, April 19 • 17:25 - 18:00
+- **[OpenTelemetry: Using Unified Semantics to Drive Insights + Project Update](https://sched.co/1HyS5)**<br>
+  by Morgan McLean, Splunk, Alolita Sharma, Apple, Daniel Dyla, Dynatrace, & Ted
+  Young, Lightstep<br> Thursday, April 20 • 16:30 - 17:05
+- **[Tutorial: Exploring the Power of OpenTelemetry on Kubernetes](https://sched.co/1HyZ3)**<br>
+  by Pavol Loffay, Benedikt Bongartz & Yuri Oliveira Sa, Red Hat, Severin
+  Neumann, Cisco & Kristina Pathak, LightStep<br> Friday, April 21 • 16:00 -
+  17:30
+
+## Co-located Events
```

Comment:
There will be as well Jaeger project booth, do you want to list it here?

Reply:
Let me think about that, because if we list jaeger, we might also consider to list other observability projects (prometheus, pixie, ...)

Reply:
not really sure how to incorporate that best, if you have any suggestions let me know, otherwise I will skip it.